src/internal/operations/test/sharepoint_test.go

```diff
@@ -177,3 +181,260 @@ func (suite *SharePointBackupIntgSuite) TestBackup_Run_sharePointExtensions() {
 		}
 	}
 }
+
+type SharePointRestoreIntgSuite struct {
+	tester.Suite
+	its intgTesterSetup
+}
+
+func TestSharePointRestoreIntgSuite(t *testing.T) {
+	suite.Run(t, &SharePointRestoreIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tconfig.M365AcctCredEnvs, storeTD.AWSStorageCredEnvs}),
+	})
+}
+
+func (suite *SharePointRestoreIntgSuite) SetupSuite() {
+	suite.its = newIntegrationTesterSetup(suite.T())
+}
+
+func (suite *SharePointRestoreIntgSuite) TestRestore_Run_sharepointWithAdvancedOptions() {
```

Comment:
it seems like much of the later part of this test could be pulled out into a helper function and used to run both OneDrive and SharePoint tests?

Reply:
very possible.  Going to save that for a patch on this PR.